CHANGELOG.md

```diff
@@ -34,9 +34,13 @@ Main (unreleased)
 
 - Flow: fix issue where Flow would return an error when trying to access a key
   of a map whose value was the zero value (`null`, `0`, `false`, `[]`, `{}`).
-  Whether an error was returned dependend on the internal type of the value.
+  Whether an error was returned depended on the internal type of the value.
   (@rfratto)
 
+- River: fix issue with evaluating expressions where if a trailing part of the 
+  expression is invalid, no error was raised. This issue does not affect Flow 
+  because it does not call ParseExpression() directly. (@ptodev)
```

Comment:
Since it's not exposed to Flow, it's not a user-facing change; can we take this out?